backend/hankkimoa/src/main/java/com/jjangiji/hankkimoa/user/controller/BookmarkController.java

```diff
@@ -0,0 +1,36 @@
+package com.jjangiji.hankkimoa.user.controller;
+
+import com.jjangiji.hankkimoa.auth.config.AuthRequiredPrincipal;
+import com.jjangiji.hankkimoa.restaurant.domain.Restaurant;
+import com.jjangiji.hankkimoa.user.domain.User;
+import com.jjangiji.hankkimoa.user.service.BookmarkService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/bookmarks")
+@RequiredArgsConstructor
+public class BookmarkController {
+    private final BookmarkService bookmarkService;
+
+    @PostMapping("/restaurants/{restaurantId}")
+    public ResponseEntity<String> addBookmark(@AuthRequiredPrincipal User user, @PathVariable Long restaurantId) {
+        bookmarkService.addBookmark(user.getId(), restaurantId);
```

Comment:
```suggestion
        bookmarkService.createBookmark(user.getId(), restaurantId);
```

컨벤션에서 CRUD로 이름 짓기로 했던 것 같아요 밑에 remove도 마찬가지로 수정이 필요해보입니다 ~